build/assets/themes/dashboard/scss/_content.scss

```diff
@@ -37,6 +37,8 @@ div#ccm-dashboard-content {
     padding-top: 1rem;
   }
 
-}
-
+  div.ccm-dashboard-content-indented {
```

Comment:
form-group elements all get a custom padding-left of 20px. Using BS4 ml-x classes we have either 16px or 24px so it doesn't align with the other stuff on the page, hence the custom class.
How do you want me to deal with that?

One possibility is to use a form-group wrapper for everything even if not dealing with a form. Would that be acceptable?

Reply:
If that's the case then other pages are going to look weird too and we can figure out how to make that consistent when we see it. I'd rather have the markup be simpler now than more complex. Honestly, I'm mostly concerned about checkboxes, input groups and buttons looking terrible because they require the new bootstrap classes